Tryouts/Prototypes/Shell/MainWindow.xaml.cs

```diff
@@ -40,13 +41,59 @@ public partial class MainWindow : Window
         private ManifestModel config;
         private ModuleModel[]? modules;
 
+        private string commandLineURL;
+        private string[] hereIsMyCode = (Application.Current as App).CustomerCode;
```

Comment:
Don't invent names. Rename to `commandLineArgs` or something that says what it does.